app/policies/portfolio_item_policy.rb

```diff
@@ -1,21 +1,40 @@
 class PortfolioItemPolicy < ApplicationPolicy
+  def create?
+    rbac_access.create_access_check
```

Comment:
@eclarizio 
Shouldn't this be
rbac_access.resource_check('update', @record.portfolio.id, Portfolio)

You can create a new PortfolioItem only if you can update the owning Portfolio. If the user has only read access to the Portfolio should not be able add a new PortfolioItem.
Part of https://projects.engineering.redhat.com/browse/SSP-1070

Reply:
@mkanoor We could definitely change it to that, but this is the same check as it was in the `before_action`, so it was probably wrong before.

Reply:
@eclarizio all those calls were incorrect, because other than Portfolio non of the other objects having sharing support. I had started a WIP PR to fix it
https://github.com/RedHatInsights/catalog-api/pull/548